packages/Telephony/Schemas/Microsoft.Telephony.CallTransfer.uischema

```diff
@@ -1,7 +1,7 @@
 ﻿{
   "$schema": "https://schemas.botframework.com/schemas/ui/v1.0/ui.schema",
   "form": {
-    "label": "Transfer a call",
+    "label": "Call Transfer",
     "subtitle": "Call Transfer",
```

Comment:
Please add a more informative subtitle. Composer shows the label in large, bold letters and the subtitle right below with lighter letter. Adding a longer sentence will much improve the user experience. Example that is not necessarily the content we want but an instance of a longer informative sentence: `Transfers a call to the provided phone number when using your bot in Telephony.`